cfme/containers/provider/openshift.py

```diff
@@ -30,7 +30,7 @@ def create(self, validate_credentials=True, **kwargs):
         # probably a js wait issue, not reproducible manually
         super(OpenshiftProvider, self).create(validate_credentials=validate_credentials, **kwargs)
 
-    def _href(self):
+    def href(self):
```

Comment:
https://docs.python.org/2/tutorial/classes.html#private-variables-and-class-local-references

It shouldn't need to change in order to access the method outside of the class. I'm not against the change - did you find that you were unable to call the method with the underscore in the name?